docs/stage-setting/stage.md

```diff
@@ -97,10 +100,100 @@ print(stage.ExportToString(addSourceFileComment=False))
 
 Here we created a `usda` file using Python, loaded it as a stage, and printed out the stage's contents. Since nothing is in our stage we do not get much from the output.
 
-`.usda` are human-readable UTF-8 text. The [Crate file](https://openusd.org/release/glossary.html#crate-file-format) format is USD's own binary file format whose file extension is `.usdc` and is bi-directionally convertible to the `.usda` text format. `.usd` can refer to either Crate or text files. 
+`.usda` are human-readable UTF-8 text. The [Crate file](https://openusd.org/release/glossary.html#crate-file-format) format is USD's own binary file format whose file extension is `.usdc` and is bi-directionally convertible to the `.usda` text format. `.usd` can refer to either Crate or text files.
```

Comment:
I'd prefer a [seealso admonition](https://github.com/NVIDIA-Omniverse/LearnOpenUSD/blob/0db2f5d2415aaec9b10f552c9311c8de3ba44c89/docs/asset-modularity-instancing/asset-modularity/what-is-asset-modularity.md?plain=1#L11) to the file formats lesson: https://docs.nvidia.com/learn-openusd/latest/stage-setting/usd-file-formats.html with some message like: "`.usda` is human-readable file format for OpenUSD. Read more about the native [file formats in OpenUSD](link/to/file/formats.md)."